src/components/MonthCalendar.tsx

```diff
@@ -0,0 +1,83 @@
+import { FormattedDate } from 'react-intl';
+
+import { ZetkinEvent } from '../types/zetkin';
+
+interface MonthCalendarProps {
+    month?: number;
+    year?: number;
+    events: ZetkinEvent[];
+}
+
+const MonthCalendar = ({ month = new Date(Date.now()).getMonth(), events, year = new Date(Date.now()).getFullYear() }: MonthCalendarProps): JSX.Element => {
```

Comment:
I think the inline default values here make this line hard to read. I think I would have just done the defaults in the body of the function. What do you think?

Optionally, maybe the argument list could be formatted differently?